### ISSUE TITLE
The 'Ensures every form element has a label' accessibility issue occurs for a Matrix Dropdown with Text cells

### ISSUE THREAD

**Issue by JaneSjs** [surveyjs/survey-library#6279]
Original issue: T13293 - Accessibility issue "Ensure every form element has a label" for matrix dropdown question 
https://surveyjs.answerdesk.io/internal/ticket/details/T13293

```
Title: WCAG 4.1.2: Ensures every form element has a label (#sq_103i)
Tags: Accessibility, WCAG 4.1.2, label

Issue: Ensures every form element has a label (label - https://accessibilityinsights.io/info-examples/web/label)

Target application: https://run.plnkr.co/preview/clibo9vex0006356s0p9maj6x/ - https://run.plnkr.co/preview/clibo9vex0006356s0p9maj6x/

Element path: #sq_103i

Snippet: <input id="sq_103i" class="sd-input sd-text" type="number" min="0" step="any" placeholder="" autocomplete="off" aria-required="false" aria-labelledby="sq_103_ariaTitle" aria-invalid="false" data-rendered="r">

How to fix: 
Fix any of the following:
  Form element does not have an implicit (wrapped) <label>
  Form element does not have an explicit <label>
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element has an empty placeholder attribute
  Element's default semantics were not overridden with role="none" or role="presentation"

Environment: Chrome version 113.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.38.1 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
```
